charts/zenko-filer/templates/deployment.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "zenko-filer.fullname" . }}
+  labels:
+    app: {{ template "zenko-filer.name" . }}
+    chart: {{ template "zenko-filer.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "zenko-filer.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+      labels:
+        app: {{ template "zenko-filer.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Values.nfsd.name }}
+          image: "{{ .Values.nfsd.image.repository }}:{{ .Values.nfsd.image.tag }}"
+          imagePullPolicy: {{ .Values.nfsd.image.pullPolicy }}
+          ports:
+            - name: nfs
+              containerPort: 2049
+          volumeMounts:
+            - name: portmap-data
+              mountPath: /run
+            - name: metadata-data
```

Comment:
would it be better to name it `metadata-conf`? Not sure just asking

Reply:
Probably yes, those were just the names that came to my mind. Do we want to change it now or later on? @jonathan-gramain